js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,12 @@
+function updateObject(...args) {
+  let indexToReplace = args[0];
+  const providedNewElement = args[1];
+  const array = args[2];
+
+  if (indexToReplace < 0 || indexToReplace >= array.length) {
```

Comment:
is the second condition necessary

Reply:
Yup that's an unnecessary condition ... will change my logic to use `splice()`  instead of using this kind of logic.